packages/big-design/src/components/Tabs/Tabs.tsx

```diff
@@ -26,12 +29,25 @@ export const Tabs: React.FC<TabsProps> = memo(
       }
     };
 
+    const activeItem = activeTab && items[Number(activeTab)];
+    const missingAriaControls = items.some((item) => !item.ariaControls);
+    const missingFallback =
+      activeItem && !document.getElementById(activeItem.ariaControls || `${activeItem.id}-content`);
+
+    if (missingAriaControls || missingFallback) {
+      warning(
+        'TabItems must have an ariaControls field, otherwise, an element with fallback ID "{tab.id}-content" must exist in the DOM',
+      );
+    }
+
     return (
       <>
         <StyledTabs {...props} flexDirection="row" role="tablist">
-          {items.map(({ id, title, disabled }) => (
+          {items.map(({ ariaControls, id, title, disabled }) => (
             <StyledTab
               activeTab={activeTab}
+              aria-expanded={id === activeTab ? 'true' : 'false'}
```

Comment:
I think this should be `aria-selected` not `aria-expanded`. Was looking at the W3's example: https://codepen.io/pen and noticed they use that instead

Reply:
Interesting, [MDN says](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role#description)

```
The tab elements not the tablist, have the [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) attribute. Set to aria-expanded="true" for the tabs associated with each visible tabpanel. The tabs associated with hidden tabpanel elements have their aria-expanded attributes set to false.
```

Reply:
But [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected#tab) they say aria-selected

Reply:
I'm going to stay with `expanded`, I did some testing with Voiceover (works with both - only notes the tab is selected/expanded), but the chrome screen reader did not work with `selected` 🤷

Reply:
Those docs might be wrong then with `aria-expanded` 🤔  I even saw on the [`role=tab`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#associated_roles_and_attributes) page that it should be `selected`